the-blue-alliance-ios/Services/RemoteConfigService.swift

```diff
@@ -0,0 +1,48 @@
+import Crashlytics
+import FirebaseRemoteConfig
+import Foundation
+
+class RemoteConfigService {
+
+    private var remoteConfig: RemoteConfig
+    internal var retryService: RetryService
+
+    init(remoteConfig: RemoteConfig, retryService: RetryService) {
+        self.remoteConfig = remoteConfig
+        #if DEBUG
+        if let configSettings = RemoteConfigSettings(developerModeEnabled: true) {
+            self.remoteConfig.configSettings = configSettings
+        }
+        #endif
+        self.retryService = retryService
+
+        remoteConfig.setDefaults(fromPlist: "RemoteConfigDefaults")
+    }
+
+    internal func fetchRemoteConfig(completion: ((_ error: Error?) -> Void)? = nil) {
+        remoteConfig.fetch(withExpirationDuration: retryInterval) { (remoteConfigStatus, error) in
+            if let error = error {
+                Crashlytics.sharedInstance().recordError(error)
+            }
+            self.remoteConfig.activateFetched()
+
+            if let completion = completion {
+                completion(error)
+            }
+        }
+    }
+
+}
+
+extension RemoteConfigService: Retryable {
```

Comment:
I'm not entirely sure if these things are worth adding in to the tests, but if we can we might as well

Reply:
Did it anyways (kinda)